CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# dbt_shopify v0.16.0
+[PR #97](https://github.com/fivetran/dbt_shopify/pull/97) includes the following updates:
+
+## Bug Fixes
+- Removed incremental logic in the following end models:
+  - `shopify__discounts`
+  - `shopify__order_lines`
+  - `shopify__orders`
+  - `shopify__transactions`
+- These models utilized the `merge` incremental strategy on BigQuery and Databricks, as we could not rely on a time series timestamp to impelment the `insert_overwrite` strategy. Using `merge` is a costly strategy, so it defeats the purpose of leveraging incremental logic. 
+- There were also concerns about the incremental logic returning incorrect data in some end models. For example, if a repeat order within the `new_vs_repeat` CTE logic in `shopify__orders` was calculated within the specified incremental window but the new order was not in that same time period, it could be incorrectly processed as a new order.  
```

Comment:
I worry this is bordering too much information that customers may not entirely understand this update. Here is a recommendation for making this a bit more succinct.

```suggestion
- Incremental strategies were removed from these models due to potential inaccuracies with the `merge` strategy on BigQuery and Databricks. For instance, the `new_vs_repeat` field in `shopify__orders` could produce incorrect results during incremental runs. To ensure consistency, this logic was removed across all warehouses. If the previous incremental functionality was valuable to you, please consider opening a feature request to revisit this approach.
```

Reply:
Updated.